src/api/test/unit/project_test.rb

```diff
@@ -18,6 +18,20 @@ def test_maintained_project_names
     assert_equal ['A', 'B', 'C'], project.maintained_project_names
   end
 
+  # TODO: bento_only
```

Comment:
where are you skipping the new UI? :confused:

Reply:
the new UI isn't using get_flags. The whole test (and the function tested) can be removed with bento - or as I wrote in the commit, we switch the old UI to the new methods as well.